### PR TITLE
event: cleanup docs slightly

### PR DESCRIPTION
**Pull request by slimsag**
Cleanup the `event` package docs slightly:

- Less verbosity in linking to MDN.
- Blank line comment for cleaner rendering on godoc.org etc.

### CODE REVIEW

File: event/generate.go [hunk 1]
```diff
@@ -161,7 +161,9 @@ func main() {
 
 	fmt.Fprint(file, `//go:generate go run generate.go
 
-// Documentation source: "Event reference" by Mozilla Contributors, https://developer.mozilla.org/en-US/docs/Web/Events, licensed under CC-BY-SA 2.5.
+// Package event defines markup to bind DOM events.
+//
+// See https://developer.mozilla.org/en-US/docs/Web/Events
```
Comment (shurcooL) [inline event/generate.go]: Nitpick. Given that this is a sentence, I'd consider not leaving out the period at the end.

Comment (neelance) [inline event/generate.go]: This change will make us violate the MDN license. Let's be nice open source citizens and do proper attribution as described on https://developer.mozilla.org/en-US/docs/MDN/About#Copyrights_and_licenses

**Comment by slimsag** [comment]
Good point @neelance and @shurcooL I hadn't considered the license aspect. I thought it was just a bad documentation line (due to `Documentation source: ...` reading strangely).

I've updated the PR with what I think would be better, PTAL.

**Comment by neelance** [comment]
LGTM

**Pull request merged**